File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseMapInfoFeatures.scala

package org.openapitools.client.model


case class ResponseMapInfoFeatures (
    _publicTransport: Option[ResponseMapInfoFeaturesPublicTransport],
    _fares: Boolean,
    _postcodes: Boolean
)
object ResponseMapInfoFeatures {
    def toStringBody(var_publicTransport: Object, var_fares: Object, var_postcodes: Object) =
        s"""
        | {
        | "publicTransport":$var_publicTransport,"fares":$var_fares,"postcodes":$var_postcodes
        | }
        """.stripMargin
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/ResponseMapInfoFeatures.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.ResponseMapInfoFeaturesPublicTransport

module ResponseMapInfoFeatures = 

  //#region ResponseMapInfoFeatures


  type ResponseMapInfoFeatures = {
    PublicTransport : ResponseMapInfoFeaturesPublicTransport;
    Fares : bool;
    Postcodes : bool;
  }
  //#endregion
  